app/workers/project_copy_worker.rb

```diff
@@ -0,0 +1,9 @@
+class ProjectCopyWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :data_high
```

Comment:
Do we want to look at locking on unique args to ensure users don't trigger multiple copies by double clicking form submission?

Reply:
✔️